cmd/podman/restore.go

```diff
@@ -29,7 +29,9 @@ var (
 			restoreCommand.GlobalFlags = MainGlobalOpts
 			return restoreCmd(&restoreCommand)
 		},
-		Example: "CONTAINER-NAME [CONTAINER-NAME ...]",
+		Example: `podman container restore ctrID
+  podman container restore --latest
+  podman container restore -all`,
```

Comment:
Missing a `-`?

Reply:
yes, fixed.  I was so worried about spaces at the end that I completely missed that.